go.mod

```diff
@@ -0,0 +1,27 @@
+module sigs.k8s.io/cluster-inventory-api
```

Comment:
> We may also need to rename this repository.

Good point. We need to update the module name then.

Reply:
Hi @dixudx @RainbowMango the general consensus from the SIG-MC community is that the repo name is independent from the ClusterProfile API name. The whole concept of "Cluster Inventory" is still fundamentally sound. It's just that unfortunately due to English spelling/grammar, it's not accurate to represent a single cluster using "ClusterInventory". Another reason is ClusterProfile might just be one of the APIs that represent the "Cluster Inventory" concept, so we shouldn't rename the repo to ClusterProfile but stick with the key concept which is still "Cluster Inventory".

Reply:
Got it. I didn't know the SIG-MC already talked about the repo name. 
It makes sense that this repo is not just for `ClusterProfile`.